lib/merchant_repository.rb

```diff
@@ -79,4 +79,8 @@ def find_invoices(id)
   def most_revenue(x)
     merchants.sort_by { |merchant| merchant.revenue }.reverse[0..(x-1)]
   end
+
+  def most_items(x)
+    merchants.sort_by { |merchant| merchant.items_sold }.reverse[0..(x-1)]
```

Comment:
Surrounding space missing for operator '-'.<br>Pass `&:items_sold` as an argument to `sort_by` instead of a block.